src/components/Individual_Article/Article.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react"
+import { getArticle } from "../../../api"
+
+const Article = ({articleId}) => {
```

Comment:
articleId should be coming from a url that is unique to that article. Not only so that I could share it with someone, but could get the appropriate data from the api with it. There is no need to be drilling this prop down.